api/internal/target/kusttarget_configplugin.go

```diff
@@ -55,6 +55,7 @@ func (kt *KustTarget) configureBuiltinTransformers(
 		builtinhelpers.LabelTransformer,
 		builtinhelpers.AnnotationsTransformer,
 		builtinhelpers.PatchJson6902Transformer,
+		builtinhelpers.ReplacementTransformer,
```

Comment:
Do you have an opinion on beginning vs end?

I think both make sense but I am leaning towards beginning. I can see someone wanting all their replacements done before any other transformer.

Reply:
I could also imagine there being arguments either way, but I was leaning towards the end, because the feature we're replacing was applied last:

> The last step of a kustomize build is to take all accumulated var configuration, build a singleton var transformer, and run it over all objects in view.

Reply:
moved it to the end, thanks!